changelog/bugfixes/2024-04-24-kernel-version-zfs-kmod.md

```diff
@@ -0,0 +1 @@
+- Fixed CI failure of zfs sysext with Kernel 6.6 by setting LINUX_INFO_BINARY_RESET=1 to prevent KV_FULL and KV_OUT_DIR from being unset. ([scripts#1951](https://github.com/flatcar/scripts/pull/1951))
```

Comment:
We don't need this entry because it's not fixing a bug of the last release.

Reply:
Done